datafusion/sql/src/planner.rs

```diff
@@ -1151,12 +1151,10 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 //
                 // For example:
                 //
-                //   SELECT c1 AS m FROM t HAVING m > 10;
```

Comment:
is there any bug for the implementation?

Reply:
currently this sql can't work

Reply:
Postgres agrees:

```sql
postgres=# create table t (c1 int);
CREATE TABLE
postgres=# insert into t  values (1), (2);
INSERT 0 2
postgres=# SELECT c1 AS m FROM t HAVING m > 10;
ERROR:  column "m" does not exist
LINE 1: SELECT c1 AS m FROM t HAVING m > 10;
                                     ^
postgres=# 
```